.github/workflows/test-environment.yml

```diff
@@ -230,6 +230,11 @@ jobs:
           aws s3 cp "${{ env.FLEET_API_DIR}}/kspm_eks.yaml" "${{ env.S3_BUCKET }}/kspm_eks.yaml"
           aws s3 cp "${{ env.FLEET_API_DIR}}/cspm-linux.sh" "${{ env.S3_BUCKET }}/cspm-linux.sh"
 
+      - name: Wait for agents to enroll
+        run: |
+          cd fleet_api
+          poetry run python src/agents_enrolled.py
```

Comment:
can we use `working-directory` instead here? see https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#defaultsrun